src/applications/static-pages/analytics/index.js

```diff
@@ -0,0 +1,28 @@
+import addJumplinkListeners from './addJumpLinkListeners';
+import addQaSectionListeners from './addQaSectionListeners';
+import addTeaserListeners from './addTeaserListeners';
+import addHomepageBannerListeners from './addHomepageBannerListeners';
+
+const pageListenersMap = new Map();
+
+pageListenersMap.set('/coronavirus-veteran-frequently-asked-questions/', [
+  addJumplinkListeners,
+  addQaSectionListeners,
+]);
+
+pageListenersMap.set('/', [addHomepageBannerListeners]);
+
+const specialListeners = pageListenersMap.get(document.location.pathname);
```

Comment:
This is super cool. Could we update the docs to specify how to add future analytic events to nodes in templates based off the pattern you're introducing here? 🙂

Reply:
Sure! I'll add a comment explaining it